### ISSUE TITLE
Suggestions for improvement

### ISSUE THREAD

**Issue by receiver1** [Amaroq7/Luna#1]
Hi, friend! The project is very interesting and I have a couple of suggestions. I suggest you use SOL2, for your own convenience. I also recommend separating spaces in the Lua language. For example, place all server functions in the “server” metatable, create a player usertype “player” and stuff it with all possible methods associated with this entity. I can also suggest using vcpkg in the project, which will make the work easier for third-party developers. And one more question: can I contact you on Telegram?

**Comment by Amaroq7** [comment]
Hi!
This project was mainly done as my university project. I'd say it more proof-of-concept project than the actual one. I have other project https://github.com/Amaroq7/SPMod/tree/test-new-meta where I planned to make universal core (which would implement different stuff, like adding cmd, timed functions, privileges) and separate each scripting language in adapter, which would make those stuff available for scripts in different languages, like Lua, SourcePawn f.e.
Sure you can contact me on Telegram, the name is the same as it is in GitHub

**Comment by receiver1** [comment]
Yes, I know about SPMod. It seems to me that the add-on to Metamod is like stuffing a sports engine into a retro car. In general, I didn’t particularly like the fact that it works Metamod. Anubis is a very cool thing for developers, with its help you can write a server entirely in C++. I hope SPMod will make a branch in which it will work on Anubis
